packages/documentation-site/patternfly-docs/content/design-guidelines/components/file-upload-multiple/file-upload-multiple.md

```diff
@@ -27,9 +27,9 @@ Most multiple file upload components contain 5 basic elements:
 
 5. **Progress upload:** The name of the uploaded file, along with file size, progress bar, and percentage.
 
-## Usage/Best practices
+## Usage
 
-**When to use:**
+### **When to use:**
```

Comment:
the "When to use" should not have a ":" at the end because it shows up in the table of contents. Should also not be bolded!